src/runtime/http.rs

```diff
@@ -0,0 +1,84 @@
+#[cfg(feature = "tokio-runtime")]
+use reqwest::{Method, Response};
+#[cfg(feature = "tokio-runtime")]
+use serde::Deserialize;
+
+#[cfg(feature = "tokio-runtime")]
+use crate::error::Result;
+
+#[derive(Clone, Debug, Default)]
```

Comment:
Rather than only defining `HttpClient` in the case that the tokio runtime is used, I instead make it an empty struct in non-tokio cases, which allows us to create it and pass it around without conditional compilation with no cost to async-std users. This helps minimize the amount of runtime-specific code we have to write outside this module.